src/main/target/REVO/target.h

```diff
@@ -136,7 +136,7 @@
 #define MPU_INT_EXTI            PC4
 #define USE_MPU_DATA_READY_SIGNAL
 
-#if !defined(AIRBOTF4) && !defined(REVOLT) && !defined(SOULF4) && !defined(PODIUMF4)
+#if defined(AIRBOTF4) || defined(AIRBOTF4SD)
```

Comment:
> airbotf4 has neither mag nor baro installed
True.

It's an example of what reconfigurable I2C/UART can do (e3fc449).
With these lines, a user can turn UART3 into I2C and connect external baro/mag.
When baro/mag is modified so that bus they are on can be specified, they can even be connected to I2C3 on PWM6/CH6.